sdk/python/tests/api/test_codegen.py

```diff
@@ -29,6 +29,7 @@ def id_map():
         "CacheID": "CacheVolume",
         "FileID": "File",
         "SecretID": "Secret",
+        "SocketID": "Socket",
```

Comment:
This shouldn't be needed since it's not being used in the tests bellow.

Reply:
:+1: will remove, I was just pattern matching here :)